packages/core/src/common/test/mock-resource-provider.ts

```diff
@@ -18,7 +19,8 @@ export class MockResourceProvider {
         return await {
             uri: new URI(''),
             dispose() { },
-            readContents(options?: { encoding?: string }): Promise<string> { return Promise.resolve(""); }
+            readContents(options?: { encoding?: string }): Promise<string> { return Promise.resolve(""); },
+            saveContentChanges(changes: TextDocumentContentChangeEvent[], options?: { encoding?: string }): Promise<void> { return Promise.resolve(); }
```

Comment:
what is `saveContentChanges` for?

Reply:
Yes, this method is optional, removed that